microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -20,147 +20,169 @@ ms.date: 05/04/2021
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# Edit files with Workspaces (Filesystem tab)
+# Edit webpage files with a DevTools workspace
 
-This tutorial provides hands-on practice in setting up and using a Workspace.  After you add files to a Workspace, the changes that you make in your source code within DevTools are saved on your local computer, and are preserved after you refresh the webpage.
+This tutorial provides hands-on practice in setting up and using a workspace in DevTools. After you add files to a workspace changes that you make in your source code with DevTools are saved on your local computer. These changes are saved  after you refresh the webpage.
 
-Before beginning this tutorial, you should know how to do the following:
-* [Use html, CSS, and JavaScript to build a web page](https://developer.mozilla.org/docs/Learn/Getting_started_with_the_web)
-* [Use DevTools to make basic changes to CSS](../css/index.md)
-* [Run a local HTTP web server](https://developer.mozilla.org/docs/Learn/Common_questions/set_up_a_local_testing_server#Running_a_simple_local_HTTP_server)
+Before starting this tutorial, you can refresh your knowledge of the technologies used by reading the following articles.
```

Comment:
I would not say you should do something else before using this page.  This page should be usable immediately by everyone.  I'd get rid of "Before..."

```suggestion
To refresh your knowledge of the technologies used, see the following articles:
```